storage/backup/backup-cli/src/backup_types/epoch_ending/restore.rs

```diff
@@ -147,9 +150,17 @@ impl EpochEndingRestoreController {
 
             // write to db
             if !lis.is_empty() {
-                self.restore_handler.save_ledger_infos(&lis)?;
-                EPOCH_ENDING_EPOCH.set(lis.last().unwrap().ledger_info().epoch() as i64);
-                EPOCH_ENDING_VERSION.set(lis.last().unwrap().ledger_info().version() as i64);
+                match self.run_mode.as_ref() {
+                    RestoreRunMode::Restore { restore_handler } => {
+                        restore_handler.save_ledger_infos(&lis)?;
+                        EPOCH_ENDING_EPOCH.set(lis.last().unwrap().ledger_info().epoch() as i64);
```

Comment:
avoid `unwrap()`, at least using `expect`?

Reply:
K. updated in the second diff in the stack. Although I think the coding guideline around absolutely no unwrap() doesn't really make sense. 🏃‍♀️